python-client/giskard/models/base/wrapper.py

```diff
@@ -99,6 +100,14 @@ def __init__(
     def _preprocess(self, data):
         if self.data_preprocessing_function:
             return self.data_preprocessing_function(data)
+
+        # In the case where a dataframe has one column, most sklearn models require a Series instead of a DataFrame.
+        # We check also the case of PredictionFunctionModel to be extra inclusive.
```

Comment:
@rabah-khalek could you add a unit test for it please?

Reply:
done!